DigitalLearningSolutions.Web/Views/SuperAdmin/Faqs/SuperAdminFaqs.cshtml

```diff
@@ -0,0 +1,44 @@
+@inject IConfiguration Configuration
+@using DigitalLearningSolutions.Web.ViewModels.SuperAdmin.Faqs
+@using Microsoft.Extensions.Configuration
+@model FaqsPageViewModel
+
+<link rel="stylesheet" href="@Url.Content("~/css/superAdmin/faqs.css")" asp-append-version="true">
+
+@{
+  ViewData["Title"] = "FAQs";
+}
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-one-quarter">
+    <nav class="side-nav-menu" aria-label="Side navigation bar">
+      <partial name="~/Views/SuperAdmin/Shared/_SideNavMenu.cshtml" model="Model" />
+    </nav>
+  </div>
+
+  <div class="nhsuk-grid-column-three-quarters">
+    <div class="nhsuk-grid-row">
+      <div class="nhsuk-grid-column-one-third">
+        <h1 id="page-heading" class="nhsuk-heading-xl">FAQs</h1>
```

Comment:
We don't need this id on the h1. It was originally used on a couple of pages, and has since persisted through copy and pastes onto new ones, but it is completed unnecessary.